espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -57,6 +58,26 @@ def add_arguments(parser):
                            help='optimizer warmup steps')
         group.add_argument('--transformer-length-normalized-loss', default=True, type=strtobool,
                            help='normalize loss by length')
+
+        E2E_rnn.loss_add_arguments(parser)
```

Comment:
This args contains an unused option `lsm-type` because of my lazy implementation (sorry).
Maybe we can simply redefine them here like the other options.
```
        group.add_argument('--ctc_type', default='warpctc', type=str,
                           choices=['builtin', 'warpctc'],
                           help='Type of CTC implementation to calculate loss.')
        group.add_argument('--mtlalpha', default=0.5, type=float,
                           help='Multitask learning coefficient, alpha: alpha*ctc_loss + (1-alpha)*att_loss ')
        group.add_argument('--lsm-weight', default=0.0, type=float,
                           help='Label smoothing weight')
```

Reply:
`lsm-type` seems to be used.

https://github.com/espnet/espnet/blob/8cbde3843ebec4d882d5ebf35aebda7ca72b7e80/espnet/nets/chainer_backend/e2e_asr.py#L65-L69

I don't know how to refactor, so please fix it yourself if necessary.